targets/nintendo-switch.json

```diff
@@ -0,0 +1,28 @@
+{
+  "llvm-target": "aarch64",
+  "build-tags": ["nintendoswitch", "arm64"],
+  "goos": "linux",
+  "goarch": "arm64",
+  "compiler": "clang",
+  "linker": "ld.lld",
+  "rtlib": "compiler-rt",
+  "libc": "picolibc",
+  "gc": "conservative",
+  "relocation-model": "pic",
+  "cflags": [
+    "-target", "aarch64-none-linux-gnu",
+    "-mtune=cortex-a53",
```

Comment:
Is there a reason we are tuning for the LITTLE core instead of the big core? This isn't necessarily a problem, it just is different from the other switch software builds I have looked at.

Reply:
Oh that's a mistake. Nintendo Switch doesn't uses the A53. It think I misplaced this. let me fix it.

Reply:
Well, it has 4 A53s and 4 A57s, but I would assume that we would want to tune for the A57s since tuning will only make a substantial difference for math-intensive workloads.

Reply:
The Horizon OS from nintendo switch keeps the A53 cores disabled all the time. There is no scheduling to it, at least anyone never has been able to reach them.

Reply:
Ah. Yeah it seems that Nintendo left them always off. . .